Sensor Analysis/sensoranalysis.py

```diff
@@ -30,8 +30,16 @@
 
 #also trim the flight data
 
+""" 
+    Aditya Srikanth 9-28-2024
+    - Removed commented code that wasn't used with sensors or unnecessarily caused errors
+    - Changed certain arr.values used in functions to dict_sensor value
+        +++ arr_df[:,7] to dict_sensor["lowG_data.timeStamp_lowG"]
+    - When figuring graph titles, isntead of keeping a long list of != statements, i made a list of excluded tags and checked to make sure element was not in the list
+"""
     
 def main():
+    sensorData = pd.read_csv("Sensor Analysis/csvfinal.csv")
```

Comment:
what's the difference between `sensorData` and `arr`? They're reading the same file